src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -86,10 +86,13 @@ function useChatUtilsHook(): UseChatUtilsHook {
 
   const goToCommunityScreen = (channel: ChannelList) => {
     setChannelAsRead(channel);
-
     const topicName = channel?.name?.replace(/^#/, '');
     const navigationParam = {
-      id: convertTopicNameToTopicPageScreenParam(topicName)
+      id: convertTopicNameToTopicPageScreenParam(topicName),
+      channelPicture: channel.channelPicture,
+      coverImage: channel.rawJson.cover_image,
+      isFollowing: channel.rawJson.is_following,
+      memberCount: channel.rawJson.topic_follower_count
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new properties added to the `navigationParam` object seem to be directly extracted from the `channel` object. This could potentially lead to runtime errors if any of these properties are undefined in the `channel` object. It would be safer to use optional chaining while accessing these properties.
```diff
-      const navigationParam = {
-       id: convertTopicNameToTopicPageScreenParam(topicName),
-       channelPicture: channel.channelPicture,
-       coverImage: channel.rawJson.cover_image,
-       isFollowing: channel.rawJson.is_following,
-       memberCount: channel.rawJson.topic_follower_count
-     };
+      const navigationParam = {
+       id: convertTopicNameToTopicPageScreenParam(topicName),
+       channelPicture: channel?.channelPicture,
+       coverImage: channel?.rawJson?.cover_image,
+       isFollowing: channel?.rawJson?.is_following,
+       memberCount: channel?.rawJson?.topic_follower_count
+     };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
mas @asaadam this recommendation would be good to implement